### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]: Jaws and NVDA are not announcing the name of the combo boxes in "Run on web app" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1461]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Tab to and select Project ->Azure->Run on web app
4. "Run on  app" dialog will open. 
5. In "Run on web app" dialog , tab to  and select "Create new web app" radio buttons.
6. Tab to Subscription combo box 
    "use Existing" combo box under resource group expand button.
    "Location and Pricing Tier" combo boxes under "App Service Plan" expand button.
    "Java Version and Artifact" combo boxes under "Java" expand button.
7. Observe the behaviour of Jaws and NVDA 

**Actual Result:**
Jaws and NVDA are not announcing the name of the combo boxes

**Jaws behaviour:**
combo box closed
blank
1 of 26
combo box closed
blank
1 of 11
combo box closed
blank
1 of 10

**NVDA Behaviour:**
combo box  collapsed
combo box  collapsed
combo box  collapsed
combo box  collapsed

**Expected Result:**
Jaws and NVDA should announce the name of the combo box
Name should be Subscription
"use Existing" combo box under resource group expand button.
"Location and Pricing Tier" combo boxes under "App Service Plan" expand button.
"Java Version and Artifact" combo boxes under "Java" expand button.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app-combo boxes-no name](https://user-images.githubusercontent.com/22956130/36101302-f341eb5c-102e-11e8-8985-55a156f45510.jpg)
![3a9a40b7](https://user-images.githubusercontent.com/22956130/36101307-f6e7920c-102e-11e8-89c6-f7c7d8f1be67.jpg)
![run on web app-combo box-explorer](https://user-images.githubusercontent.com/22956130/36101311-f940ffe8-102e-11e8-8433-a63b40cc5436.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749931
